### ISSUE TITLE
[Design System Toolbox] Remove use of CocoaPods

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-ios#1103]
# Context

_CocoaPods_ will be sunset soon. More details in their [blog publication](https://blog.cocoapods.org/CocoaPods-Specs-Repo/).

# Description

We use this dependency manger in the _design system toolbox_ project, thus it should be updated.

# Definition of Ready

- [x] Maybe not too much branches or pull requests on going to prevent boring rebases

# Definition of Done

- [x] Update GitHub Actions workflows to remove `pod install` command
- [x] Update internal GitLab CI pipelines to remove `pod install` command
- [x] Remove CocoaPods gem from Gemfile, and update the lock
- [x] Update the THIRD-PARTY file
- [x] Update DEVELOP and any documentation refering to `pod install` command
- [x] Maybe remove the .xcworkspace if the Package.resolved file can be store elsewhere (default is inside)

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.22.0**
- Build number: **1762898813**
- Build tag: **9176741**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.22.0**
- Build number: **1763503643**
- Build tag: **15e1f58**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.22.0**
- Build number: **1763590034**
- Build tag: **187365f**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New release available** 🚀

Please, get the [new release from GitHub](https://github.com/Orange-OpenSource/ouds-ios-design-system-toolbox/releases/tag/0.22.0) and wait for App Store update to test it!

- Display name: **Design System Toolbox**
- Version: **0.22.0**
- Build number: **1764339495**



_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
